rustls/src/client/builder.rs

```diff
@@ -82,6 +75,37 @@ impl ConfigBuilder<ClientConfig, WantsTransparencyPolicy> {
         )))
     }
 
+    /// Sets a single certificate chain and matching private key for use
```

Comment:
The duplication of these methods is a little ugly, but I think this does provide the most ergonomic API for callers.